posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12504 @@
+{
```

Comment:
<p>Same thing here - just a quick overview of dask + xarray with sparse arrays, or feel free to remove the "works"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>